Code/GraphMol/MolStandardize/test1.cpp

```diff
@@ -37,8 +37,8 @@ void testCleanup() {
   }
   {
     // Github 5997
-    auto m = "CC(=O)O[Mg]OC(=O)C"_smiles;
-    auto res(MolStandardize::cleanup(*m, params));
+    RWMOL_SPTR m = "CC(=O)O[Mg]OC(=O)C"_smiles;
```

Comment:
I don't think this should be necessary. The _smiles literal returns a unique_ptr

Reply:
Right, I got carried away a bit.

Reply:
I figured that you were just checking to see if I was paying attention 😉_